backend/python/app/services/implementations/story_service.py

```diff
@@ -96,6 +96,16 @@ def import_story(self, details, file):
             self.logger.error(str(error))
             raise error
 
+    def process_story(self, story_file):
+        try:
+            story_contents = self._read_doc(story_file["path"])
```

Comment:
the only thing this function does is call `_read_doc`, can we instead just rename `_read_doc->process_story` and pass in the correct path fromo the process_story gql query?